plugins/services/src/js/utils/__tests__/TaskTableUtil-test.js

```diff
@@ -9,13 +9,14 @@ describe('TaskTableUtil', function () {
   beforeEach(function () {
     this.getServiceHealth = MarathonStore.getServiceHealth;
     MarathonStore.getServiceHealth = function (prop) {
-      return this[prop].get('health');
+      return this[prop]['health'];
```

Comment:
@Poltergeist I agree with you that we shouldn't have inconsistencies. However, as @mlunoe points out - we've never enforced structs for tasks. In fact IMO, our structs (at least how they currently work via `Item.get('property')` causes far more complexity than was intended. We will be deprecating structs with Relay/GraphQL. Let's keep what @weblancaster has proposed as this will follow a concept that will persist (i.e. no structs).